distrax/_src/distributions/gumbel.py

```diff
@@ -108,7 +110,7 @@ def variance(self) -> Array:
 
   def mode(self) -> Array:
     """Calculates the mode."""
-    return self._loc * jnp.ones_like(self._scale)
+    return self._loc
```

Comment:
I think we should return `self.loc` (i.e., without the underscore) so that the shape matches with the `batch_shape +  event_shape` of the distribution.